packages/protoc-gen-connect-query/src/generateDts.ts

```diff
@@ -12,26 +12,54 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import type { DescFile, DescService } from '@bufbuild/protobuf';
-import { codegenInfo, MethodKind } from '@bufbuild/protobuf';
-import type { Schema } from '@bufbuild/protoplugin';
-import { localName } from '@bufbuild/protoplugin/ecmascript';
+import type { DescFile, DescService } from "@bufbuild/protobuf";
+import { codegenInfo, MethodIdempotency, MethodKind } from "@bufbuild/protobuf";
+import type { Schema } from "@bufbuild/protoplugin";
+import {
+  literalString,
+  localName,
+  makeJsDoc,
+} from "@bufbuild/protoplugin/ecmascript";
 
-import type { PluginInit } from './utils';
+import type { PluginInit } from "./utils";
 
 const { safeIdentifier } = codegenInfo;
 
+// prettier-ignore
 /**
  * Handles generating a TypeScript Declaration file for a given Schema, DescFile (protobuf definition) and protobuf Service.
  */
 const generateServiceFile =
   (schema: Schema, protoFile: DescFile) => (service: DescService) => {
+    const { MethodKind: rtMethodKind, MethodIdempotency: rtMethodIdempotency } = schema.runtime;
+    
     const f = schema.generateFile(
       `${protoFile.name}-${localName(service)}_connectquery.d.ts`,
     );
 
     f.preamble(protoFile);
 
+    f.print(makeJsDoc(service));
+    f.print("export declare const ", localName(service), ": {");
```

Comment:
Just a bit of copy pasting. Once we prioritize one plugin, we can just delete all of it. That's a diff PR.